Samples/HelloWorld/Tests/HelloWorldSiloTests.cs

```diff
@@ -40,7 +40,7 @@ public static void ClassCleanup()
             // Optional. 
             // By default, the next test class which uses TestignSiloHost will
             // cause a fresh Orleans silo environment to be created.
-            StopAllSilos();
+            StopAllSilosIfRunning();
```

Comment:
Nit.  Odd naming.  Isn't it assumed we'd only stop running silos?  If we want to be explicit about them running in the call name, StopAllRunningSilos would seem less clumsy.